symengine/tests/test_functions.py

```diff
@@ -88,12 +88,31 @@ def test_Subs():
 
 def test_FunctionWrapper():
     import sympy
-    sympy.var("n, m, theta, phi")
+    n, m, theta, phi = sympy.symbols("n, m, theta, phi")
     r = sympy.Ynm(n, m, theta, phi)
     s = Integer(2)*r
     assert isinstance(s, Mul)
     assert isinstance(s.args[1]._sympy_(), sympy.Ynm)
 
+    x = symbols("x")
+    e = x + sympy.loggamma(x)
+    assert str(e) == "x + loggamma(x)"
```

Comment:
Also add more tests to exercise all the methods like +, -, / that you implemented.